src/errors/test-run/index.js

```diff
@@ -48,14 +48,26 @@ class ActionArgumentErrorBase extends ActionError {
 
 // Synchronization errors
 //--------------------------------------------------------------------
-export class MissingAwaitError extends TestRunErrorBase {
+export class SynchronizationError extends TestRunErrorBase {
```

Comment:
Base classes should be in `Base` section: https://github.com/AlexanderMoskovkin/testcafe-phoenix/blob/5d9a3e870a3eaca01241efb0e8257425eab64b6e/src/errors/test-run/index.js#L9